docs/how-to/heterogeneous_matches.rst

```diff
@@ -0,0 +1,64 @@
+.. _heterogeneous-matches:
+
+Use custom matches
+<<<<<<< HEAD
+===================
+=======
+===================
+>>>>>>> a90487a68b0b2997e860d6735dea0b52deece66b
```

Comment:
```suggestion
===================
```

Reply:
You created a marge conflict when you merged: f887da4. These "lines"  were added by git to show you where the conflict was happening 👍🏻

Reply:
Thanks for letting me know! :)